swiper.el

```diff
@@ -514,6 +529,28 @@ such as `scroll-conservatively' are set to a high value.")
   "A predicate that decides whether `line-move' or `forward-line' is used.
 Note that `line-move' can be very slow.")
 
+(defun swiper--format-function (candidates)
+  "Format function for swiper CANDIDATES.
+
+Highlight current line number as in `display-line-numbers-mode'."
+  (ivy--format-function-generic
+   (lambda (str)
+     (let ((ln (if swiper-include-line-number-in-search str
+                 (copy-sequence (get-text-property 0 'display str)))))
+       (when (and ln
+                  (eq (get-text-property 0 'face ln)
+                      'swiper-line-number-face))
+         (put-text-property 0 (next-single-property-change 0 'face
+                                                           ln (length ln))
```

Comment:
Why is `(length ln)` necessary? Why can't you just write `(next-single-property-change 0 'face ln)`?